### DISCUSSION TITLE
Additional cargo types

### DISCUSSION THREAD

**Discussion by Coryf88** [Neefay/BromA-Framework-A3-Addon#190] [Ideas]
Discuss and suggest additional [cargo types](https://github.com/broma-arma/BromA-Framework-A3-Addon/blob/master/loadouts/cargo-list.sqf) to add.

**Comment by Royallle** [comment]
"toolkit" adds a toolkit to a vehicle.
"rangetables" adds the various range tables of ACE, or have one type for each range table
"contraband" adds ciggies, lolipops and cigars
"entrenching_tools" adds entrenching tools
"wirecutters" adds wirecutters

would be interesting to have "at_mines" and "ap_mines" however these should be related to the loadouts unlike the "mines" we currently have that uses US ones.

**Comment by rundabollen** [comment]
Ammo for ACE CSW would be good: HMG, Mortar, and static AT.

**Comment by Coryf88** [comment]
Weapons? "rifle" would currently add ammo for the `_commonRIFLE`, could change that, and the others, to add the weapon itself and instead do like "rifle_ammo" for it's ammo.

**Reply by Royallle** [reply]
Don't know about that, this wouldn't be that common and I guess you can keep it this way since it would make sense to add the gun and ammo for it.

**Reply by Coryf88** [reply]
Yeah, it's use would be rare, but having the option for it would be handy and simplify those edge cases.

Also, this would be even better, since it'd give all combinations:
"rifle" - Weapon + ammo
"rifle_weapon" - Weapon only
"rifle_ammo" - Ammo only

**Reply by Coryf88** [reply]
Though, one could just do ["rifle_weapon", "rifle_ammo"] for the "rifle", but I'd personally like "rifle" to be an option for something, preferably the ammo since it'd most likely be the most used and the weapon be "rifle_weapon". 🤔
Though, "rifle" being the weapon makes more sense.

**Reply by rundabollen** [reply]
"riflekit" = "rifle"+"rifleammo"?
